src/almanack/metrics/metrics.yml

```diff
@@ -59,6 +59,13 @@ metrics:
     description: >-
       Boolean value indicating the presence of a CITATION file
       or some other means of indicating how to cite the work.
+  - name: "default-branch-not-master"
```

Comment:
I'm wondering why this is a metric - are we making a value judgement on the name of the head reference?

Reply:
Thanks for this question! My guesstimate was that when cloning the repo that the default branch would be the head reference. I'm digging in to test this assumption out a bit and try to see if there's a way to find the default branch when working from a dev branch.

Reply:
I ended up adding a check for the remote head and sought a remote `master` reference from there as this is a common pattern I found when looking through older GitHub repos.